herbie/settings.py

```diff
@@ -0,0 +1,205 @@
+"""
+Django settings for herbie project.
+
+Generated by 'django-admin startproject' using Django 2.2.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+import environ
+
+env = environ.Env()
+
+# Business entity application label
+APP_LABEL = 'herbieapp'
+
+# Json schema package for validation of business objects
+SCHEMA_REGISTRY_PACKAGE = 'schema'
+
+# chunk size for exporting data
+DEFAULT_CHUNK_SIZE = 100
+
+# message_provider
+MESSAGING_PROVIDER = 'google_pubsub'
```

Comment:
Do we still need this variable?

Reply:
I think we can remove this and the logic assigned to it. Will double check